tasks/human_eval_056.rs

```diff
@@ -9,7 +9,53 @@ use vstd::prelude::*;
 
 verus! {
 
-// TODO: Put your solution (the specification, implementation, and proof) to the task here
+spec fn spec_bracketing_helper(brackets: Seq<char>) -> (int, bool) {
```

Comment:
Thank you for spotting this. in the case match for `>`, it was supposed to be `x != 0` instead of `x==0`. Its tracking if x ever goes below zero, which is the criteria for incorrect bracketing. I've fixed this now